src/libraries/System.Runtime.InteropServices/gen/LibraryImportGenerator/Analyzers/ConvertToLibraryImportAnalyzer.cs

```diff
@@ -50,7 +49,7 @@ public override void Initialize(AnalysisContext context)
                 context =>
                 {
                     // Nothing to do if the LibraryImportAttribute is not in the compilation
-                    INamedTypeSymbol? libraryImportAttrType = context.Compilation.GetBestTypeByMetadataName(TypeNames.LibraryImportAttribute);
+                    INamedTypeSymbol? libraryImportAttrType = context.Compilation.GetTypeByMetadataName(TypeNames.LibraryImportAttribute);
```

Comment:
i think you still want GetBestTypeByMetadataName.  That's a useful helper that solves important problems like when tehre may be multiple types in metadata with this name.

Reply:
Is that API present in some other version of Roslyn? Or is that just not in? I thought the plan was to, if possible, not have to ship Roslyn bits as part of the generators that ship inbox.

Reply:
No, the `GetBestTypeByMetadataName` is not a public Roslyn API and is not planned to be. The definition of "Best" is nebulous, so Roslyn doesn't want to prescribe the policy there. However, the runtime teams and Roslyn have the same idea of what "best" means in this case, so we use this helper. We should keep this helper.